etc/freertos.mips4k.pic32mx.mk

```diff
@@ -55,11 +55,12 @@ ASFLAGS = -c -g -EL -MD -MP $(BASEDEFS) -D__LANGUAGE_ASSEMBLY__ -fdollars-in-ide
 #           -march=armv7-m -mthumb -mfloat-abi=soft
 
 CORECFLAGS = -c -EL -g -msoft-float -march=mips32r2 $(ARCHOPTIMIZATION) \
-	     -Wall -Werror -Wno-unknown-pragmas -MD -MP \
```

Comment:
The warnings are coming from "pic32\include\peripheral\adc10.h". Not sure what to do with them, since the whole header contains just a bunch of irrelevant #defines (I think).

#if !defined( _DISABLE_OPENADC10_CONFIGPORT_WARNING )
    #warning "Support for configport argument for OpenADC10 not available for this device."
  #endif

 #if !defined( _DISABLE_OPENADC10_CONFIGSCAN_WARNING )
    #warning "ADC Channels 32-50 not supported for configscan OpenADC10 argument for this device."
 #endif